hacks_mbof/settings.py

```diff
@@ -164,7 +164,7 @@
             'level': 'DEBUG',
         },
         'django.db.backends': {
-            # 'level': 'DEBUG', # Uncomment to log all SQL statements
+            'level': 'DEBUG', # Uncomment to log all SQL statements
```

Comment:
Can the value of 'level' be a variable and be injected, maybe as an environment variable?  Needing to change code to change debug level is a pain.

Maybe settings.py should be treated like a properties file, so a single build can just be configured to point to different versions.